numpy/doc/subclassing.py

```diff
@@ -555,6 +555,46 @@ def __array_wrap__(self, arr, context=None):
 how this can work, have a look at the ``memmap`` class in
 ``numpy.core``.
 
+Subclassing and Downstream Compatibility
+----------------------------------------
+
+When sub-classing ``ndarray`` or creating duck-types that mimic the ``ndarray``
+interface, it is your responsibility to decide how aligned your APIs will be
+with those of numpy. There is sometimes great temptation and confusion amongst
+users of libraries downstream from numpy to use numpy methods on downstream
+objects that may actually not be compatible with numpy's methods.
+
+For example, if you want your sub-class or duck-type to be compatible with
+numpy's ``sum`` method, the method signature for this object's ``sum`` method
```

Comment:
To minimize confusion, instead of "numpy's `sum` method" I would write "the `np.sum` function".